transit_odp/fares_validator/schema/fares_schema.json

```diff
@@ -23,6 +33,490 @@
+      "reference": "0",
+      "number": 131,
+      "context": "//x:FareFrame/x:salesOfferPackages",
+      "rules": [
+        {
+          "test": "count(x:SalesOfferPackage) > 0"
+        }
+      ]
+    },
+    {
+      "details": "Mandatory element 'distributionAssignments' missing",
+      "category": "Mandatory Element",
+      "reference": "0",
+      "number": 131,
+      "context": "//x:salesOfferPackages/x:SalesOfferPackage",
+      "rules": [
+        {
+          "test": "count(x:distributionAssignments) > 0"
+        }
+      ]
+    },
+    {
+      "details": "Mandatory element 'DistributionAssignment' missing",
+      "category": "Mandatory Element",
+      "reference": "0",
+      "number": 131,
+      "context": "//x:salesOfferPackages/x:SalesOfferPackage/x:distributionAssignments",
+      "rules": [
+        {
+          "test": "count(x:DistributionAssignment) > 0"
+        }
+      ]
+    },
+    {
+      "details": "Mandatory element 'DistributionChannelType' missing",
+      "category": "Mandatory Element",
+      "reference": "0",
+      "number": 131,
+      "context": "//x:salesOfferPackages/x:SalesOfferPackage/x:distributionAssignments/x:DistributionAssignment",
+      "rules": [
+        {
+          "test": "count(x:DistributionChannelType) > 0"
+        }
+      ]
+    },
+    {
+      "details": "Mandatory element 'PaymentMethods' missing",
+      "category": "Mandatory Element",
+      "reference": "0",
+      "number": 131,
+      "context": "//x:salesOfferPackages/x:SalesOfferPackage/x:distributionAssignments/x:DistributionAssignment",
+      "rules": [
+        {
+          "test": "count(x:PaymentMethods) > 0"
+        }
+      ]
+    },
+    {
+      "details": "Mandatory element 'salesOfferPackageElements' missing",
+      "category": "Mandatory Element",
+      "reference": "0",
+      "number": 131,
+      "context": "//x:salesOfferPackages/x:SalesOfferPackage",
+      "rules": [
+        {
+          "test": "count(x:salesOfferPackageElements) > 0"
+        }
+      ]
+    },
+    {
+      "details": "Mandatory element 'SalesOfferPackageElement' missing",
+      "category": "Mandatory Element",
+      "reference": "0",
+      "number": 131,
+      "context": "//x:salesOfferPackages/x:SalesOfferPackage/x:salesOfferPackageElements",
+      "rules": [
+        {
+          "test": "count(x:SalesOfferPackageElement) > 0"
+        }
+      ]
+    },
+    {
+      "details": "Mandatory element 'TypeOfTravelDocumentRef' missing",
+      "category": "Mandatory Element",
+      "reference": "0",
+      "number": 131,
+      "context": "//x:salesOfferPackages/x:SalesOfferPackage/x:salesOfferPackageElements/x:SalesOfferPackageElement",
+      "rules": [
+        {
+          "test": "count(x:TypeOfTravelDocumentRef) > 0"
+        }
+      ]
+    },
+    {
+      "details": "Mandatory element 'PreassignedFareProductRef' missing",
+      "category": "Mandatory Element",
+      "reference": "0",
```

Comment:
Please remove reference and number from the schema definition from all the places